src/compiler/diagnosticMessages.json

```diff
@@ -4849,6 +4849,22 @@
         "category": "Error",
         "code": 6238
     },
+    "Using cached result of 'package.json' at '{0}' that indicates it was found.": {
+        "category": "Message",
+        "code": 6239
+    },
+    "Using cached result of 'package.json' at '{0}' that indicates it was not found.": {
+        "category": "Message",
+        "code": 6240
+    },

```

Comment:
@DanielRosenwasser do you have suggestion for text here.. this seems really odd and not very clear immediately if you found package.json or not/.
Its in trace and i think we should mention that it was result from cache so that its hard to diagnose.

Reply:
What about

```
File '{0}' exists according to earlier cached lookups.
```

and

```
File '{0}' does not exist according to earlier cached lookups.
```